src/App.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```

Comment:
- [ ] Please, consider fixing your README file - it comes from CRA and needs to be modified considering the [template from MICROVERSE](https://github.com/microverseinc/readme-template/blob/master/README.md) for making your project more professional 👍